app/controllers/add-preprint.js

```diff
@@ -11,6 +12,13 @@ export default Ember.Controller.extend({
         uploadMultiple: false,
         method: 'PUT'
     },
+    canEdit: Ember.computed('isAdmin', 'isRegistration', function() {
+        return this.get('isAdmin') && !(this.get('model').get('registration'));
```

Comment:
This seems like a question we ask a lot, and might make sense to add helper methods on the model class.